artman/cli/main.py

```diff
@@ -286,6 +295,12 @@ def normalize_flags(flags, user_config):
         flags.config = os.path.join(flags.root_dir, flags.config)
     else:
         flags.config = os.path.abspath(flags.config)
+    try:
+        _validate_config_flag(flags.config)
```

Comment:
It looks like this parses the config, and then throws it away. Why are we parsing it twice?

Reply:
It looks like this parses the config, and then throws it away. Why are we parsing it twice?

Reply:
Good point. Moved the logic to loader.py which is a better place to do the validation under current implementation.